### PR TITLE
django-configurations 1.0

### PR DESCRIPTION
**Pull request by blueyed**

### CODE REVIEW

File: generate_configurations.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ def requirements(env):
     yield 'pytest==%s' % (env.pytest_version)
     yield 'pytest-xdist==1.13.1'
     yield DJANGO_REQUIREMENTS[env.django_version]
-    yield 'django-configurations==0.8'
+    yield 'git+https://github.com/jezdez/django-configurations'
```
Comment (ticosax) [inline generate_configurations.py]: `s/jezdez/jazzband/` :wink:

Comment (blueyed) [inline generate_configurations.py]: Thanks! :)
(but is redirected currently and will be replaced by 1.0 soonish.. ;)

Comment (jezdez) [inline generate_configurations.py]: :heart_eyes_cat:

**Comment by blueyed** [comment]
Included in #314.

**Pull request closed without merging**